manifest/build.go

```diff
@@ -6,11 +6,16 @@ import (
 	"strings"
 )
 
-func (m *Manifest) Build(dir, appName string, s Stream, cache bool) error {
+func (m *Manifest) Build(targetService, dir, appName string, s Stream, cache bool) error {
```

Comment:
Can we factor out a BuildService instead? This params list is getting rather unwieldy.